packages/net/net_address.pony

```diff
@@ -108,18 +110,69 @@ class val NetAddress is Equatable[NetAddress]
       recover String.from_cstring(consume serv) end)
 
   fun eq(that: NetAddress box): Bool =>
-    (this.length == that.length)
-      and (this.family == that.family)
-      and (this.port == that.port)
+      (this._family == that._family)
+      and (this._port == that._port)
       and (host_eq(that))
-      and (this.scope == that.scope)
+      and (this._scope == that._scope)
 
   fun host_eq(that: NetAddress box): Bool =>
     if ip4() then
-      this.addr == that.addr
+      this._addr == that._addr
     else
-      (this.addr1 == that.addr1)
-        and (this.addr2 == that.addr2)
-        and (this.addr3 == that.addr3)
-        and (this.addr4 == that.addr4)
+      (this._addr1 == that._addr1)
+        and (this._addr2 == that._addr2)
+        and (this._addr3 == that._addr3)
+        and (this._addr4 == that._addr4)
+  end
```

Comment:
The `end` here should be the same level of indentation as the `if` that it terminates. That is, it should be indented one more level to the right.